### PR TITLE
Added pelops dataset for STR labeled SLiCE dataset

### PR DESCRIPTION
**Pull request by skpelops2017**
Dataset subclass and test module for SLiCE chips labeled by STR.

### CODE REVIEW

File: pelops/datasets/slice.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        except Exception as e:
+            sys.stderr.write(str(e))
+
+    def __create_chip(self, file_info, truth_value):
+        """Converts parsing / indexing results into a pelops.datasets.chip.Chip object"""
+
+        try:
+            if truth_value == 0:
+                self.__noise_seq += 1
+                car_id = 'unk-%.9d' % self.__noise_seq
```
Comment (agude) [inline pelops/datasets/slice.py]: Prefer `.format()` to `%`.

File: pelops/datasets/slice.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    def __set_chips(self):
+        """Sets the chips dict of the superclass to contain chip files for the dataset."""
+
+        # Scan filesystem
+        root_files = [root_file for root_file in os.walk(self.dataset_path)]
+
+        # Decode truth.txt file
+        truth_files = [os.path.join(walked[0], 'truth.txt') for walked in root_files if 'truth.txt' in walked[2]]
+        if len(truth_files) == 0:
+            raise Exception("No truth file found.")
```
Comment (agude) [inline pelops/datasets/slice.py]: There is probably a more specific error to raise. Maybe `IOError`?

File: testci/test_slice.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import shutil
+import pytest
+import tempfile
+import pelops.datasets.slice as slice
+
+
+@pytest.fixture
+def slice_env():
+    """Setup mock STR SLiCE dataset"""
+    work_dir = tempfile.mkdtemp('_pelops_testing')
```
Comment (agude) [inline testci/test_slice.py]: Prefer pytest's built in `tmpdir` fixtures: https://docs.pytest.org/en/latest/tmpdir.html

**Comment by skpelops2017** [comment]
First round of corrections committed, CI tests passed.

**Comment by agude** [comment]
Looks good!

**Pull request merged**